### PR TITLE
[BuildBot] Uplift GPU RT version for Linux CI Process

### PR DESCRIPTION
**Pull request by bb-sycl**
Uplift GPU RT version for Linux to 22.11.22682

### CODE REVIEW

File: devops/dependencies.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
       "root": "{DEPS_ROOT}/tbb/lin"
     },
     "oclcpu": {
-      "github_tag": "2021-WW50",
-      "version": "2021.13.11.0.23",
-      "url": "https://github.com/intel/llvm/releases/download/2021-WW50/oclcpuexp-2021.13.11.0.23_rel.tar.gz",
+      "github_tag": "2022-WW13",
```
Comment (bader) [inline devops/dependencies.json:28]: @alexbatashev, this PR is supposed to update GPU driver, not CPU. There is another PR for updating CPU runtime.

**Comment by alexbatashev** [comment]
> @alexbatashev, all tests pass. Why did you rolled back CPU driver version?

the above comment from you asks to do so

**Comment by bader** [comment]
> > > @alexbatashev, all tests pass. Why did you rolled back CPU driver version?
> > 
> > 
> > the above comment from you asks to do so
> 
> I commented before #5883 had been merged and I was asking to keep CPU driver unmodified. That is what I implemented myself with [626beed](https://github.com/intel/llvm/commit/626beedc46f54a715a2682eca9ea00ba14866e95). You commits revert #5883. Please, revert [97feb6b](https://github.com/intel/llvm/commit/97feb6ba5c8af0c566cb03723ac8a8ad8186470a) and [02a1159](https://github.com/intel/llvm/commit/02a11597db103c90303dbe5267f9c9697fa5108a).

@alexbatashev, ping.

**Comment by bader** [comment]
BTW, maybe we can drop this PR and focus on https://github.com/intel/llvm/pull/5952.

**Review by bader — APPROVED** [review]

**Comment by cperkinsintel** [comment]
closing as no longer relevant.

**Pull request closed without merging**